react/features/virtual-background/functions.ts

```diff
@@ -61,14 +59,14 @@ export const toDataURL = async (url: string) => {
  * @returns {Promise<string>}
  *
  */
-export function resizeImage(base64image: any, width: number = 1920, height: number = 1080): Promise<string> {
+export function resizeImage(base64image: any, width = 1920, height = 1080): Promise<string> {
 
     // In order to work on Firefox browser we need to handle the asynchronous nature of image loading;  We need to use
     // a promise mechanism. The reason why it 'works' without this mechanism in Chrome is actually 'by accident' because
     // the image happens to be in the cache and the browser is able to deliver the uncompressed/decoded image
     // before using the image in the drawImage call.
     return new Promise(resolve => {
-        const img = document.createElement('img');
+        const img = document.createElement('img') as any;
```

Comment:
as HTMLImageElement?

Reply:
HTMLImageElement was the inferred type. The problem is that a few lines below we have `context.drawImage(img..` and that errors because it doesn't support HTMLImageElement. Instead of adding `@ts-ignore` I just added `as any` here